common/dynamicconfig/constants.go

```diff
@@ -524,6 +524,11 @@ is currently processing a task.
 		1000,
 		`NexusEndpointListMaxPageSize is the maximum page size for listing Nexus endpoints.`,
 	)
+	NexusPayloadSizeLimit = NewNamespaceIntSetting(
```

Comment:
We don't need another dynamic config. It should be the same as the other payload limit in the system and should not get out of sync.